<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceOrderWindow</class>
 <widget class="QDialog" name="PlaceOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Place order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="6,1,1">
   <item row="0" column="0">
    <widget class="QTableWidget" name="cartTableWidget"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="totalPriceLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="placeOrderButton">
     <property name="text">
      <string>Place order</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
